packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -408,10 +413,35 @@ export const contract = async (
     makeOpenPortfolioInvitation() {
       trace('makeOpenPortfolioInvitation');
       return zcf.makeInvitation(
-        (seat, offerArgs) => {
+        async (seat, offerArgs) => {
           mustMatch(offerArgs, offerArgsShapes.openPortfolio);
+          await null;
           consumeAccessToken(seat);
-          return openPortfolio(seat, offerArgs);
+
+          const kit = makeNextPortfolioKit();
+
+          const publicSubscribers: PublicSubscribers = {
+            portfolio: {
+              description: 'Portfolio',
+              // getStoragePath() is a vow for async flow membrane but we know it will resolve promptly
+              storagePath: await vowTools.asPromise(
+                kit.reader.getStoragePath(),
+              ),
+              subscriber: null as any,
+            },
+          };
+          // This flow does its own error handling and always exits the seat
+          void openPortfolio(
```

Comment:
If registering the noble forwarding account fails, `openPortfolio` used to fail rather than providing a broken portfolio. I think the `rebalance should fail when noble account creation fails` was supposed to test this, though the intent is poorly communicated.

Options
 - register the NFA before calling `openPortfolio`
 - punt to a future issue
 - ... other?